src/core/actions/categories.js

```diff
@@ -1,13 +1,15 @@
 import {
   CATEGORIES_FETCH,
   CATEGORIES_LOAD,
-  CATEGORIES_FAIL,
 } from 'core/constants';
 
-export function categoriesFetch() {
+export function categoriesFetch({ errorHandlerId } = {}) {
+  if (!errorHandlerId) {
```

Comment:
> When we use Flow 100% we won't need it. 

^ That was a hypothetical statement to illustrate how we can only be sure that our app is calling action creators in the right way with something like Flow or with if statements that check every required variable for truthiness. Take your pick. 

Personally, I think this is an excellent example of how Flow can help us. If our app calls an action creator incorrectly it could be really hard to track down since that information is completely lost in the trail of dispatched actions (we have no trail of function calls to action creators).